xun/tests/test_dask_driver.py

```diff
@@ -14,3 +14,4 @@ def test_dask_driver():
         result = blueprint.run(driver=dask_driver, store=store)
 
     assert result == expected
+    client.close(0.1)
```

Comment:
0.1?

Reply:
It worked with just closing. I have just read a recommendation to put there some small time offset just to be sure.

Reply:
done